### ISSUE TITLE
ng-serve refresh browser when changes in scripts folder but changes are not applied

### ISSUE THREAD

**Issue by aestheticsdata** [angular/angular-cli#8347]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
Angular CLI: 1.5.0
Node: 7.8.0
OS: darwin x64
Angular: 5.0.0
... animations, common, compiler, compiler-cli, core, forms
... http, platform-browser, platform-browser-dynamic
... platform-server, router, upgrade

`@angular/cli: 1.5.0`
`@angular-devkit/build-optimizer: 0.0.32`
`@angular-devkit/core: 0.0.20`
`@angular-devkit/schematics: 0.0.35`
`@ngtools/json-schema: 1.1.0`
`@ngtools/webpack: 1.8.0`
`@schematics/angular: 0.1.0`
typescript: 2.4.2
webpack: 3.8.1

### Repro steps.
updating angular-cli 1.0.0 to 1.5.0

$ ng-serve

change in app.js script which path is defined in .angular-cli.json

### The log given by the failure.
Browser si refreshed but changes in app.js are not updating in browser. Is there some changes in the way angular-cli 1.5.0 serves watched files ?

### Desired functionality.
Would like to have an updated doc of angular-cli 1.5.0

### Mention any other details that might be useful.
the desired behavior was working correctly with angular 4 and angular-cli 1.0.0
maybe related issue : https://github.com/angular/angular-cli/issues/8313

**Comment by Brocco** [comment]
I can confirm that the page indeed refreshes, but the new changes in script files are not applied.

**Comment by paragmeshram** [comment]
Yes, this is not working for me as well. Need a workaround

**Comment by clydin** [comment]
This should be corrected in 1.6.3.  If the problem is still occurring, please re-open the issue.
